sql/moz-fx-data-shared-prod/firefoxdotcom_derived/ga_sessions_v1/bigconfig.yml

```
@@ -0,0 +1,37 @@
+type: BIGCONFIG_FILE
+tag_deployments:
+- collection:
+    name: Operational Checks
+    notification_channels:
+    - slack: '#de-bigeye-triage'
+  deployments:
+  - column_selectors:
+    - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.firefoxdotcom_derived.ga_sessions_v1.submission_date
+    - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.firefoxdotcom_derived.ga_sessions_v1.ga_client_id
+    - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.firefoxdotcom_derived.ga_sessions_v1.ga_session_id
+    metrics:
+    - saved_metric_id: is_not_null
+      rct_overrides:
+      - submission_date
```

```suggestion
      - session_date
```
Looking at the corresponding metadata this appears to be the field used for partitioning.